packages/FormElement/src/components/Content/Content.styles.js

```diff
@@ -0,0 +1,12 @@
+import styled, { css } from "styled-components";
+
+const inlineContentStyles = css`
+  align-items: baseline;
+  display: flex;
+`;
+
+export const Content = styled.div(
+  ({ isInline }) => css`
+    ${isInline && inlineContentStyles};
+  `
```

Comment:
I think the `Content` component could be agnostic of this `isIinline` prop (which should be set only on the parent `FormElement`).  The styling for FormElement could then apply any transformation CSS necessary to both the Label and the Content (but maybe that could even be avoided if it simply wraps them both in a flex layout).